.github/workflows/docker.yml

```diff
@@ -170,3 +252,13 @@ jobs:
           org.opencontainers.image.licenses=${{ github.event.repository.license.spdx_id }}
         cache-from: type=local,src=/tmp/.buildx-cache
         cache-to: type=local,dest=/tmp/.buildx-cache
+    - name: Slack Notification
+      if: ${{ steps.prep.outputs.slack-notification-version && steps.prep_base_version.outputs.should-notify-slack == 'true' && job.status == 'success' }}
```

Comment:
On `main.yml` I used `if github_repository.lower() == 'hathornetwork/hathor-core':` to check test if this is the main repo or a fork. Should we do something similar to decide whether to enable this step? Otherwise if it's running on a fork this step will just fail.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/547/commits/e018be00b4c0152945686c6de7890b54a56bf157 (This link will eventually break since I need to rebase the branch frequently, which is one of the reasons I would rather use merges inside of PRs)